### DISCUSSION TITLE
Slow slider update and clickable knob granularity

### DISCUSSION THREAD

**Discussion by scrollop** [rsjaffe/MIDI2LR#606] [Problems]
Hello,
midi2lr works REALLY well with a Behringer x-touch compact, and it's quite easy to use. A couple of questions:

 1) Using motorized sliders is a bit slow. For example, it takes 4 seconds to change exposure from 5 to -5 (you can see the exposure value reducing on screen). 

2) Is it possible to change the granularity of clickable knobs, so instead of one click being, say, 0.5 of a crop, it is less? In options you can change temperature/exposure maximum values which can alter this, though it's not available for other variables.

Thanks a lot for creating this great software!

**Comment by bad1tv** [comment]
Hi!
I have same problem
Can you fix it?
